pkg/common/common.go

```diff
@@ -160,6 +160,14 @@ const (
 	PublishContextFCWWPNPrefix = "FCWWPN"
 	// WWNPrefix indicates WWN prefix
 	WWNPrefix = "naa."
+	// SyncMode indicates Synchronous Replication
+	SyncMode = "SYNC"
+	// AsyncMode indicats Asynchronous Replication
+	AsyncMode = "ASYNC"
+	// Zero indicates value zero for RPO
+	Zero = "Zero"
+	// Metro indicates Metro mode
+	Metro = "METRO"
```

Comment:
'MetroMode' - to be consistent with other mode names? Or an enum that defines the modes.